src/contracts/verifier/contract/src/lib.rs

```diff
@@ -99,26 +102,34 @@ impl VerifierProxy {
                 // bytes is a vector of length 32, where the last bit is 1|0 depending on the truthy value
                 // Parse only the last bit and use that to determine if this is true or false
                 let is_valid = bytes.get(31).unwrap().clone() == 1;
-                // if this proof is invalid then throw an error
-                if !is_valid {env::panic_str("message")};
+
+                env::log_str("bytes: {bytes}");
```

Comment:
Does this make it into production contracts, or does the optimiser for mainnet deployment remove this?

Reply:
not at all, this will be reverted